ci/files/containers.yaml

```diff
@@ -0,0 +1,17 @@
+---
+# tcib container images used by cloudops team for telemetry testing
+container_images:
+- imagename: quay.io/podified-master-centos9/openstack-aodh-api:current-podified
```

Comment:
Why are these using quay.io/podified-master-centos9 instead of quay.rdoproject.org/podified-master-centos10?

Reply:
I have copied these names from tcib default container config https://github.com/openstack-k8s-operators/tcib/blob/main/container-images/containers.yaml, these urls are overridden by tcib tool while building the containers.